sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateAsyncClientTest.java

```diff
@@ -514,7 +501,7 @@ public void cancelCertificateOperation(HttpClient httpClient, CertificateService
 
             StepVerifier.create(certPoller
                     .takeUntil(asyncPollResponse ->
-                        asyncPollResponse.getStatus() == LongRunningOperationStatus.USER_CANCELLED)
+                        "cancelled".equalsIgnoreCase(asyncPollResponse.getStatus().toString()))
```

Comment:
revert?

Reply:
No, the service returns "cancelled" instead of "user_cancelled" and has for quite some time. I don't know why the tests never caught this, but I went back to some old recordings and found "cancelled" was the value used. I'll check some very old recordings just in case.